electionleaflets/apps/api/views.py

```diff
@@ -17,7 +17,7 @@ class StandardResultsSetPagination(LimitOffsetPagination):
 class LeafletFilter(filters.FilterSet):
     class Meta:
         model = Leaflet
-        fields = ["date_uploaded"]
+        fields = {"date_uploaded": ["gt", "exact"]}
```

Comment:
`gte`?

Reply:
I think it needs to be `gt` because we get the timestamp of the last object in WCIVF. So I think using `gte` would mean it keeps importing the last object